Source/DafnyCore/Verifier/Translator.ExpressionWellformed.cs

```diff
@@ -712,28 +712,52 @@ private record ReadsCheckDelayer(ExpressionTranslator etran, Function selfCallsA
                 }
               }
               // check that the preconditions for the call hold
-              foreach (AttributedExpression p in e.Function.Req) {
```

Comment:
What's driving the changes at this location? I thought most of what you had to do was add extra attributes.